web/assets/js/dashboard.js

```diff
@@ -182,22 +182,19 @@ function loadTableData(tableTarget, sourceData, colTitle) {
     var tableId = tableTarget.attr('id');
     tableTarget.html("<table class='table table-striped table-condensed raw-data-table'><thead><tr id='" + tableId + "-headers'><th>" + colTitle + "</th></tr></thead><tbody id='" + tableId + "-body'></tbody></table>");
     var headerRow = $('#' + tableId + '-headers');
-    $(sourceData[0].x).each(function(i, date) {
+    $(sourceData[0]['x']).each(function(i, date) {
         headerRow.append($("<th>").text(date));
     });
 
     // load scores
     var tableBody = $('#' + tableId + '-body');
     $($(sourceData).get().reverse()).each(function(index, row) {
-        // Ignore row if no data provided
```

Comment:
This was inadvertently reverted as well (fixing a bug where an RDR value that's expected (but not yet available) comes through as an empty row..